### ISSUE TITLE
type *zip.Writer has no field or method Flush

### ISSUE THREAD

**Issue by michardy** [edgi-govdata-archiving/zip-starter#9]
When I attempt to run go build in accordance with the instructions in the readme I get:
``` bash
~/Code/zip-starter$ go build
# _/home/michael/Code/zip-starter
./server.go:120: zw.Flush undefined (type *zip.Writer has no field or method Flush)
```
Is there a dependency or something I am missing? (I have golang installed)

**Comment by b5** [comment]
Hi @michardy! I suspect it may be that you're running a previous version of go, what is the output of `go version` on your machine? Updating to go 1.8 or later should solve the problem.

**Comment by michardy** [comment]
Thanks. I fixed the problem by upgrading from 1.3.3 to 1.7.  It now compiles.
This is the third edgi project that Debian has broken due to out of date packages.
